core/src/main/kotlin/links/DRI.kt

```diff
@@ -23,6 +26,26 @@ data class DRI(
     }
 }
 
+object EnumEntryDRIExtra: DRIExtraProperty<EnumEntryDRIExtra> {
+    override val key = "EnumEntry"
+}
+
+interface DRIExtraProperty<T> {
```

Comment:
i think it should also have: `@JsonTypeInfo(use = CLASS)` as this will avoid confusion between different entries with the same structure.